app/views/users/index.html.erb

```diff
@@ -1,14 +1,16 @@
 <h1>Here is the list of Users</h1>
 <% @users.each do |user| %>
-<article class='user'>
-  <figure>
-    <img class='user-image' src= '<%= user.photo %>' alt="<%= user.name %>" />
-  </figure>
-  <%= link_to user_path(user.id) do  %>
-    <section class='user-info'> 
-      <h2>User: <%= user.name %></h2>
-      <p>Number of post:<strong> <%= user.post_counter || 0 %> </strong> </p>
-    </section>
-  <% end %>
+  <article class='user'>
+    <figure>
+      <img class='user-image' src= '<%= user.photo %>' alt="<%= user.name %>" />
+    </figure>
+    <%= link_to user_path(user.id) do  %>
+      <section class='user-info'> 
+        <h2>User: <%= user.name %></h2>
+        <p>Number of post:<strong> <%= user.post_counter || 0 %> </strong> </p>
+      </section>
+    <% end %>
 </article>
 <% end %>
+<%= link_to "Add a post", new_user_post_path(@current_user), class: 'button w120' %>
```

Comment:
- Hey, here's this error thrown from this line. Please, kindly try to fix it. After this, your app should still meet the project requirements.


![image](https://github.com/LarryIVC/larrys-blog/assets/59397159/ad693089-2c10-4de4-8cf1-579eb093e41b)

Reply:
Hi @zieeco, this is because there are no users and the method returns a null.